community/consistency-check/src/main/java/org/neo4j/consistency/ConsistencyCheckService.java

```diff
@@ -173,53 +182,48 @@ public Result runFullConsistencyCheck( final File storeDir, Config tuningConfigu
             }
         } ) );
 
+        // Bootstrap kernel extensions
+        Dependencies dependencies = new Dependencies();
+        dependencies.satisfyDependencies( consistencyCheckerConfig, fileSystem,
+                new SimpleLogService( logProvider, logProvider ), IndexStoreView.EMPTY, pageCache );
+        KernelContext kernelContext = new SimpleKernelContext( fileSystem, storeDir, UNKNOWN, dependencies );
+        LifeSupport life = new LifeSupport();
+        KernelExtensions extensions = life.add( new KernelExtensions(
+                kernelContext, (Iterable) load( KernelExtensionFactory.class ), dependencies, ignore() ) );
+
         try ( NeoStores neoStores = factory.openAllNeoStores() )
         {
-            LabelScanStore labelScanStore = null;
-            try
-            {
-                IndexStoreView indexStoreView = new NeoStoreIndexStoreView( LockService.NO_LOCK_SERVICE, neoStores );
-                OperationalMode operationalMode = OperationalMode.single;
-                labelScanStore = new LuceneLabelScanStoreBuilder( storeDir, indexStoreView, fileSystem,
-                        consistencyCheckerConfig, operationalMode, logProvider )
-                        .build();
+            life.start();
+            IndexStoreView indexStoreView = new NeoStoreIndexStoreView( LockService.NO_LOCK_SERVICE, neoStores );
```

Comment:
Actually this is a mistake, the IndexStoreView should be used as the dependency into the kernel extensions. However it's not reasonable that CC should be able build the label scan store from store on startup to check consistency of it later, but that's apparently how the code did. I don't think we have a single test which starts a CC on a store w/o label scan store though.

I can restore the old behaviour, i.e. using those dependencies again.